contrib/completion/bash/docker

```diff
@@ -70,6 +85,57 @@ __docker_container_ids() {
 	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
 }
 
+__docker_images() {
+	local images_args=""
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_IDS" in
+		all)
+			images_args="--no-trunc -a"
+			;;
+		non-intermediate)
+			images_args="--no-trunc"
+			;;
+	esac
+
+	local awk_script=''
+
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_ID" in
+		all)
```

Comment:
This is what happens when you knock out a change at 5PM on a Friday. I'll
get that fixed up on Monday. Thanks for the feedback!

On 8 August 2015 at 13:27, Harald Albers <notifications@github.com> wrote:

> In contrib/completion/bash/docker
> <https://github.com/docker/docker/pull/15332#discussion_r36577802>:
>
> > @@ -70,6 +85,57 @@ __docker_container_ids() {
> >  	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
> >  }
> >
> > +__docker_images() {
> > +	local images_args=""
> > +	case "$DOCKER_COMPLETION_SHOW_IMAGE_IDS" in
> > +		all)
> > +			images_args="--no-trunc -a"
> > +			;;
> > +		non-intermediate)
> > +			images_args="--no-trunc"
> > +			;;
> > +	esac
> > +
> > +	local awk_script=''
> > +
> > +	case "$DOCKER_COMPLETION_SHOW_IMAGE_ID" in
> > +		all)
>
> As the all and non-intermediate branches are identical, they should be
> combined.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/15332/files#r36577802>.
>



-- 
Rory Hunter